tests/devices/qubit_mixed/test_qubit_mixed_measure.py

```diff
@@ -68,54 +60,12 @@ def test_sample_based_observable(self, mp, two_qubit_state):
             _ = measure(mp, two_qubit_state)
 
 
-class TestMeasurementDispatch:
```

Comment:
Since the new `default.mixed` device has a `get_measurement_function`, why are we deleting all these tests?

Shouldn't we just update them if this is incompatible with the new device API?